scheduler/src/cook/mesos/util.clj

```diff
@@ -167,22 +167,34 @@
 
 (timers/deftimer [cook-mesos scheduler get-pending-jobs-duration])
 
+(defn get-pending-job-ents*
```

Comment:
I'm okay with the name, I like the convention of `*` suffixes to ask users to use the function with care.